miner/miner.go

```diff
@@ -325,7 +326,9 @@ minerLoop:
 
 			if err := m.sf.MinedBlock(b.Header, base.TipSet.Height()+base.NullRounds); err != nil {
 				log.Errorf("<!!> SLASH FILTER ERROR: %s", err)
-				continue
+				if os.Getenv("LOTUS_MINER_NO_FILTER") != "_yes_i_know_i_can_and_probably_will_lose_all_my_fil_and_power_" {
```

Comment:
Perhaps?

```suggestion
				if os.Getenv("LOTUS_MINER_NO_SLASHFILTER") != "_yes_i_know_i_can_and_probably_will_lose_all_my_fil_and_power_" {
```

W have a number of other `FILTER`s